Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -528,23 +528,18 @@ public void onResult(@Nullable DropInResult dropInResult, @Nullable Exception er
                 } else {
                     final DropInResult dropInResult = new DropInResult();
                     dropInResult.paymentMethodNonce(paymentMethodNonce);
-
-                    if (mDropInRequest.getCollectDeviceData()) {
-                        getDropInClient().collectDeviceData(DropInActivity.this, new DataCollectorCallback() {
-                            @Override
-                            public void onResult(@Nullable String deviceData, @Nullable Exception error) {
-                                if (deviceData != null) {
-                                    dropInResult.deviceData(deviceData);
-                                    animateBottomSheetClosedAndFinishDropInWithResult(dropInResult);
-                                } else {
-                                    updateVaultedPaymentMethodNonces(true);
-                                    onError(error);
-                                }
+                    getDropInClient().collectDeviceData(DropInActivity.this, new DataCollectorCallback() {
+                                                                          @Override
+                                                                          public void onResult(@Nullable String deviceData, @Nullable Exception error) {
```

Comment:
Nitpick - but the indentation looks off here for the `override` and `public void` lines

Reply:
Ah yeah those get collapsed in the IDE so didn't notice the weird indent. Should be fixed now.